terraform/deployments/govuk-publishing-platform/.terraform.lock.hcl

```diff
@@ -1,6 +1,27 @@
 # This file is maintained automatically by "terraform init".
```

Comment:
Accidental commit?

Reply:
Terraform advices that we include [dependency lock files](https://www.terraform.io/docs/language/dependency-lock.html) (a recent addition to Terraform) in version control partly for code review:

> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.

Reply:
Yuck. Fair enough, let's go with their recommendation then :)